spec/mailers/csv_submitted_for_processing_first_stage_email_mailer_spec.rb

```diff
@@ -16,12 +16,12 @@
     let(:template_ids) do
       {
         in_progress: "in-progress-uuid",
```

Comment:
Should the `pending` status be here too?
```suggestion
        pending: "pending-uuid",
```